pixoo/__init__.py

```diff
@@ -245,6 +247,24 @@ def draw_text(self, text, xy=(0, 0), rgb=Palette.WHITE):
         for index, character in enumerate(text):
             self.draw_character(character, (index * 4 + xy[0], xy[1]), rgb)
 
+    def draw_character2(self, character, xy=(0, 0), rgb=Palette.WHITE):
+        matrix = retrieve_glyph2(character)
+        if matrix is not None:
+            teiler = matrix[-1]
+            for index, bit in enumerate(matrix):
+                if bit == 1:
+                    local_x = index % teiler
+                    local_y = int(index / teiler)
+                    self.draw_pixel((xy[0] + local_x, xy[1] + local_y), rgb)
+
+    def draw_text2(self, text, xy=(0, 0), rgb=Palette.WHITE):
+        for index, character in enumerate(text):
+            if index == 0:
+                matrix = 0
+            self.draw_character2(character, (matrix + xy[0], xy[1]), rgb)
+            matrix += retrieve_glyph2(character)[-1] + 1
+
+
```

Comment:
This is duplicated code.
A more generic approach for `draw_character` and `draw_text` would be better.

Reply:
as also in line 453 I forgot to remove the comments from the programming... i am still quite a beginner but i give my best :D

Reply:
basically I find the idea very good maybe you can simply select the font by parameter I try it at time to implement.

Reply:
ok iam Done!
now u can via `from pixoo import Pixoo, SimulatorConfig, FONT_GICKO, FONT_PICO_8 `
include the different Fonts 
in this example u have the my font FONT_GICKO
`pixoo.draw_text('1337', (0, 0), (255, 255, 255), FONT_GICKO)`